FloripaPublicTransportationTests/RestApiTests.swift

```diff
@@ -0,0 +1,343 @@
+//
+//  RestApiTests.swift
+//  FloripaPublicTransportation
+//
+//  Created by Marcelo Gobetti on 2/23/16.
+//  Copyright © 2016 Marcelo Gobetti. All rights reserved.
+//
+
+import XCTest
+@testable import FloripaPublicTransportation
+import OHHTTPStubs
+
+class RestApiTests: XCTestCase, ExpectationProtocol {
```

Comment:
Hi @arctouch-pedrocosta , I am attaching the zipped .plist summary of the tests. I have just cloned my repository and checked out the release/1.0 branch, then without having ever built the project and without any kind of modifications to it, I have clicked the little "play" button in the RestApiTests and the tests succeeded normally, as you can check in the attached file. Thanks!
[AD26155A-4720-43B7-BEC2-09C3CE09DB59_TestSummaries.plist.zip](https://github.com/gobetti/FloripaPublicTransportation/files/157302/AD26155A-4720-43B7-BEC2-09C3CE09DB59_TestSummaries.plist.zip)